sonic_installer/main.py

```diff
@@ -480,7 +480,9 @@ def sonic_installer():
 @sonic_installer.command('install')
 @click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
               expose_value=False, prompt='New image will be installed, continue?')
-@click.option('-f', '--force', is_flag=True,
```

Comment:
>@click.option('-f', '--force [](http://example.com/codeflow?start=0&length=28)

You can use
```
@click.option('-f', '--force', 'force_secureboot', ...)
```

To keep old option string and have a good python var name.

Reply:
However `force_secureboot` is not accurate. The only use case is the installation path secure->non-secure. So the best var name should be "skip_secure_check"